Communication/DaemonCommander.cs

```diff
@@ -1,58 +1,58 @@
 ﻿using System;
 using System.Text;
-using System.Net;
-using System.Net.Sockets;
+using System.Net.Http;
 using System.Text.Json;
-using System.Text.Json.Serialization;
-using System.Windows.Forms;
+using System.Threading.Tasks;
+using HttpOverStream.NamedPipe;
 
 namespace CRCTray.Communication
 {
 
 	static class DaemonCommander
 	{
-		static private string socketPath = string.Format("{0}\\.crc\\crc.sock", Environment.GetFolderPath(Environment.SpecialFolder.UserProfile));
+		private static readonly string _daemonHTTPNamedPipe = "crc-http";
+        private static readonly string _apiPath = "/api";

```

Comment:
Not sure why the indentation jumps around

![image](https://user-images.githubusercontent.com/1894/123654923-2048c780-d861-11eb-8523-c8c160bdd061.png)